public/schema.json

```diff
@@ -52,6 +59,12 @@
       }
     },
     "schemas": {
+      "Account": {
```

Comment:
Should have 3 here for accounts:
- BasicAccount
- BasicAccounts
- Account

Can you add the last one as a schema - that would belong in the `account/:id` endpoint - currently it is hard coded but would be better that it is defined in a schema